shared/actions/tracker.js

```diff
@@ -426,157 +427,177 @@ const sessionIDToUsername: { [key: number]: string } = {}
 function serverCallMap (dispatch: Dispatch, getState: Function, skipPopups: boolean = false, onFinish: ?() => void): incomingCallMapType {
   return {
     'keybase.1.identifyUi.start': ({username, sessionID, reason}, response) => {
+      response.result()
```

Comment:
we respond immediately and defer the work that is a side effect of the incoming message